src/main/migrations/changesets/wdl_and_inputs.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <property name="clob.type" value="LONGTEXT" dbms="mysql"/>
```

Comment:
@scottfrazer and I discussed this a while back and I forget what we decided. Certainly that `LONGTEXT` would never be exceeded :) IIRC the discussion was if the smaller one would 100% be safe or if we should use the medium sized one. 

AFAICT there's no real danger to upgrading if it turns out 64K isn't enough for everybody, to paraphrase a famous underestimator of memory usage.